lib/matplotlib/lines.py

```diff
@@ -790,6 +790,9 @@ def draw(self, renderer):
                     cap = self._dashcapstyle
                     join = self._dashjoinstyle
                 else:
+                    print(tpath.vertices.size)
+                    tpath = affine.inverted().transform_path(affine.transform_path(tpath).cleaned(simplify=True))
+                    print(tpath.vertices.size)
```

Comment:
Yeah, it looks like `renderer` is basically the backend. Without requiring changes to the backend API (so that it takes in an already transformed path, and I'm sure it was not designed that way initially also for performance reasons), I don't think we can avoid it. It's still faster than *not* doing the downsampling.